src/components/URLInput/URLInput.js

```diff
@@ -5,16 +5,14 @@ import Button from "@material-ui/core/Button";
 import { makeStyles } from "@material-ui/core";
 import "./URLinput.css";
 import URLInputHeader from "./URLInputHeader";
-import Tooltip from "@material-ui/core/Tooltip";
```

Comment:
I think there must have been a merge issue somewhere. This is the Tooltip for the URLinput, so it definitely needs to be here.